src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,38 @@
+package racingcar;
+
+public class Car {
+
+    private String name;
+    private int position;
+
+    public Car(String name) {
+        this.name = name;
+        this.position = Constant.START_POSITION;
+    }
+
+    private int makeRandom() {
+        return (int) (Math.random() * Constant.RANDOM_RANGE);
+    }
+
+    public boolean isMovable() {
+        return makeRandom() >= Constant.MINIMUM_MOVE;
+    }
+
+    public void movePosition() {
+        if (isMovable()) {
+            position++;
+        }
+    }
```

Comment:
저희가 글을 읽을 때에 위에서 아래로 읽듯 코드도 같은 방향으로 읽죠..!

그렇기에 최상위의 함수인 `movePosition`을 가장 위에 두고, 그 함수에서 사용하는 `isMovable`을 다음, 그 함수에서 사용하는 `makeRandom`이라는 함수를 마지막에 두면 좋을 것 같아요.

첫 함수를 읽고 내부에서 사용되는 함수가 있구나 생각하고 그 함수가 다음에 바로 나오면 혼란스럽지 않을테니까요.

지금의 방식대로라면 내부에서 사용되는 함수에 대한 컨텍스트를 머리에 담고 이를 사용하는 함수를 읽고, 다시 이를 사용하는 함수를 읽어야해서 뇌가 저장해야하는 정보가 많아진달까요?!

```suggestion
    public void movePosition() {
        if (isMovable()) {
            position++;
        }
    }
    
    // Car 클래스에서만 사용하기에 private으로 바꿔도 되겠죠?
    private boolean isMovable() {
        return makeRandom() >= Constant.MINIMUM_MOVE;
    }
    
    private int makeRandom() {
        return (int) (Math.random() * Constant.RANDOM_RANGE);
    }
```